estate/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+{
+    'name': "Real Estate",
+    'summary': """
+        Testing the real estate module
+    """,
+
+    'description': """
+        Testing the description real estate module
+    """,
+    'depends': ['base'],
+    'data': [
+        'security/ir.model.access.csv',
+        'views/estate_property_view.xml',
+        'views/estate_menus.xml'
+    ],
+    'application': True,
+    'installable': True,
+    'author': "Odoo",
+    'license': 'AGPL-3'
```

Comment:
```suggestion
    'license': 'LGPL-3',
```
1. There are two kind of Odoo licenses: `LGPL-3` that is for the Open-Source part of Odoo (Community), and `OEEL-1` for the Enterprise part.
2. Don't forget to put trailing comma at the end of a dict. It's a good practice to keep the git history cleaner: if a colleague needs to add a value to the dict, he can add it at the end without the need to modify your code.